src/main/java/sortBench/algorithms/MergeSort.java

```diff
@@ -0,0 +1,50 @@
+package sortBench.algorithm ;
+
+public class MergeSort implements ISortingAlgorithm {
+     
+    public void sort(int inputArr[]) {
+        this.array = inputArr;
```

Comment:
Ok test are failing because you need to define class attributes.
All variables prefix by `this.` are class attributes.

Below the third line (class definition) add simply lines below:
```
private int[] array;
private int length;
private int[] tempMergArr;
```

Reply:
Say I haven't been figuring it out !
Well, updated anyway. Is it going fine now ?

Reply:
I did not see your change. Did you push it ?

Reply:
You may have some issues with your fork history there is a lot of merge between your branches.
![image](https://user-images.githubusercontent.com/3628660/59716074-8a134d00-9215-11e9-98ec-b59e8f611cc1.png)

Reply:
I've got it as commit 9c330e4.
Kinda weird... Trying to reupload it.

I'm all okay with the history. That's but there's been a lot of differences in concept between what Git users call "master" and what I do... Hence somewhat of a fusion as if into a 'local master'. (Which wasn't much use anyway, except for Explorer-related purposes, so I just ended up leaving them as I dived down further into those separate fixes.)

Reply:
It should be okay now.